linux/settings.env

```diff
@@ -17,4 +17,4 @@ export OMERODIR=/opt/omero/server/OMERO.server
 # Location of the virtual environment for omero-py
 VENV_SERVER=/opt/omero/server/venv3
 
-export PATH=$VENV_SERVER/bin:$PATH
+export PATH=$VENV_SERVER/bin:/opt/rh/rh-python38/root/usr/bin:$PATH
```

Comment:
One consideration while looking again to this. Since the entire installation/deployment process is encapsulated using Python virtual environment, I think it is possible that updating the environment `PATH` to always include the new Python environment is not strictly necessary. At minimum, the `python3 -mvenv` command in `step01_centos7_ice_venv.sh` needs to know about the Python 3.8 path but I do not think this is a required later on.

This is driven while thinking of equivalent changes at the Ansible level where some RH7 specific handling could be introduced in https://github.com/ome/ansible-role-python3-virtualenv/blob/947d49c596fa5fee3318d8c8d7d5b4b4bd3f9899/files/ome-python3-virtualenv.sh#L19

Reply:
I oped to add i to ``settings.env`` since our doc we indicate to run ``source settings.env`` so making the change will not require an export PATH before running the command

Reply:
Another option if we don't want to export the PATH would be to explicitly set the path to the executable in the CentOS 7 file e.g.

```
/opt/rh/rh-python38/root/usr/bin/python -m venv ...
```

In addition to the point above, my other concern is whether this will appear in Ubuntu walkthroughs as well as it might create confusion

Reply:
that will also change the doc. I understood we wanted to keep ``python3 -m venv..``